.travis.yml

```diff
@@ -10,12 +10,18 @@ before_install:
 install:
 -  npm install
 -  npm install -g grunt-cli
+-  pip install .
 -  pip install -r requirements_dev.txt
 -  pip install coveralls
 script:
+# Test that we can run from source directory
 - bin/kalite start --traceback -v 2
 - bin/kalite status
 - bin/kalite stop --traceback -v 2
+# Test that we can run as the installed script
```

Comment:
What is this testing? Instead of calling all three commands again, can we just call one command (like `status`) since `start` is pretty slow.